torchtune/modules/transforms/tokenizers/__init__.py

```diff
@@ -4,6 +4,7 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+from ._hf_tokenizer import HFTokenizer
```

Comment:
nit: it's weird that we spell out SentencePiece and TikToken, but not HuggingFace...

Reply:
OK yeah we can spell it out